rerun_py/tests/unit/test_box3d.py

```diff
@@ -69,15 +71,15 @@ def test_boxes3d() -> None:
         half_sizes = half_sizes if half_sizes is not None else half_sizes_arrays[-1]
 
         # make Pyright happy as it's apparently not able to track typing info trough zip_longest
-        half_sizes = cast(Vec3DArrayLike, half_sizes)
-        centers = cast(Vec3DArrayLike, centers)
-        rotation_axis_angles = cast(RotationAxisAngleArrayLike, rotation_axis_angles)
-        quaternions = cast(QuaternionArrayLike, quaternions)
-        radii = cast(Optional[Float32ArrayLike], radii)
-        colors = cast(Optional[Rgba32ArrayLike], colors)
-        labels = cast(Optional[Utf8ArrayLike], labels)
-        class_ids = cast(Optional[ClassIdArrayLike], class_ids)
-        fill_mode = cast(Optional[rr.components.FillMode], fill_mode)
+        half_sizes = cast("Vec3DArrayLike", half_sizes)
+        centers = cast("Vec3DArrayLike", centers)
+        rotation_axis_angles = cast("RotationAxisAngleArrayLike", rotation_axis_angles)
+        quaternions = cast("QuaternionArrayLike", quaternions)
+        radii = cast("Optional[Float32ArrayLike]", radii)
+        colors = cast("Optional[Rgba32ArrayLike]", colors)
+        labels = cast("Optional[Utf8ArrayLike]", labels)
+        class_ids = cast("Optional[ClassIdArrayLike]", class_ids)
+        fill_mode = cast("Optional[rr.components.FillMode]", fill_mode)
```

Comment:
same (+ many more....)

Reply:
Side thought: we should figure out a better way for these archetype tests, it's annoying to have to create/maintain them manually, and I don't like the implementation (e.g. these annoying casts).